packages/xo-server/src/api/schedule.mjs

```diff
@@ -64,3 +66,26 @@ delete_.params = {
 }
 
 export { delete_ as delete }
+
+export async function runSequence({ idSchedules }) {
+  const t = new Task({ properties: { type: 'xo:schedule:sequence', name: 'Schedule sequence' } })
+  await t.run(async () => {
+    const nb = idSchedules.length
+    const signal = Task.abortSignal
+    for (let i = 0; i < nb; i++) {
+      if (signal.aborted) {
+        throw new Error(signal.reason)
+      }
+      Task.set('progress', Math.round((i * 100) / nb))
+      const idSchedule = idSchedules[i]
+      const schedule = await this.getSchedule(idSchedule)
+      const job = await this.getJob(schedule.jobId)
+      await this.runJob(job, schedule)
+    }
+  })
+}
+runSequence.permission = 'admin'
+runSequence.description = 'Run a sequence of schedule, one after the other'
+runSequence.params = {
+  idSchedules: { type: 'array', items: { type: 'string' } },
```

Comment:
IMHO, `schedules` is more in line with the rest of the API (contrary to `job.runSequence` which accept a very badly named `idSequence` array of ids :exploding_head:).